### PR TITLE
Correct PassiveScanControllerUnitTest name

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by ricekot** [comment]
The build is failing.

**Comment by kingthorin** [comment]
@psiinon seems some tests are failing

**Comment by kingthorin** [comment]
@psiinon Got plans for this? 😉

**Comment by psiinon** [comment]
Ugh, yeah, I need to revisit this.
Last time I looked I could not work out why it was happening .. will look again...

File: zap/src/test/java/org/zaproxy/zap/extension/pscan/PassiveScanControllerUnitTest.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
 import org.zaproxy.zap.utils.I18N;
 
 /** Unit test for {@link PassiveScannerController}. */
```
Comment (thc202) [inline zap/src/test/java/org/zaproxy/zap/extension/pscan/PassiveScanControllerUnitTest.java:47]: `PassiveScanController`

**Comment by thc202** [comment]
Seems to be a timing issue, not failing always locally.

**Review by thc202 — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Thank you both

**Comment by psiinon** [comment]
Thanks @thc202 !

**Pull request merged**